Makefile

```diff
@@ -1128,12 +1123,10 @@ qmail-log.5
 qmail-lspawn: \
 load qmail-lspawn.o spawn.o prot.o slurpclose.o coe.o sig.a wait.a \
 case.a cdb.a fd.a open.a stralloc.a substdio.a error.a str.a \
-fs.a auto_qmail.o uid.o gid.o auto_userp.o auto_userq.o auto_groupn.o \
-auto_spawn.o
+fs.a ids.a auto_qmail.o uid.o gid.o
```

Comment:
Nitpick: `ids.a` after `gid.o`

Reply:
I tried to keep groups of archives or object files together whereever possible, even if that means I shift these things around a bit. If we go for the .o solution I would squash the last commit into the first one and entirely get rid of the intermediate state, so this will not persist.